lib/kubernetes-deploy/deploy_task.rb

```diff
@@ -255,6 +255,11 @@ def validate_resources(resources)
       KubernetesDeploy::Concurrency.split_across_threads(resources) do |r|
         r.validate_definition(kubectl, selector: @selector)
       end
+
+      resources.select(&:has_warnings?).each do |r|
```

Comment:
style nit: `r` -> `resource` (or at least something that isn't a single letter var for a multi-line block).

Reply:
I'll fix it in this one place. Fixing it all over the file will make it harder to review for others. In a smaller PR or a separate one I can pick at cleaning it up everywhere.